scss/foundation-styles/buttons.scss

```diff
@@ -83,39 +83,39 @@
   }
 }
 
-// append external link glyph
-.btn-secondary[href*="//"]:not([href*="foundation.mozilla.org"]) {
-  display: inline-flex;
-  align-items: center;
-
-  &::after {
-    filter: brightness(0);
-    content: " ";
-    display: block;
-    width: 16px;
-    height: 14px;
-    background: url(../_images/glyphs/external.svg) no-repeat;
-    margin-left: 8px;
-    position: relative;
-    bottom: 1px;
-    transition: filter 0.2s ease-in-out;
-  }
-
-  @include hover-focus-active {
-    &::after {
-      filter: brightness(1);
-    }
-  }
-
-  @at-root .dark-theme & {
-    &::after {
-      filter: brightness(1);
-    }
-
-    @include hover-focus-active {
-      &::after {
-        filter: brightness(0);
-      }
-    }
-  }
-}
+// // append external link glyph
+// .btn-secondary[href*="//"]:not([href*="api.mozillapulse.org"]) {
```

Comment:
This attempted to append a glyph to the SignIn / SignOut button on mobile nav when you're logged out. What would be the correct url to add here to avoid this so I can uncomment these lines 😅 ?

Reply:
maybe we should toggle `signupin` class on the mobile SignIn / SignOut button based on user's login status?

Reply:
So we can't remove this entire chunk of SCSS since we will need it for the "Get involved" button on the entry page (see mockup of the new design). Since Pulse is not powered by CMS like the Foundation site, I think we can do this in a more manual way to avoid crazy complicated SCSS selector

Change 
```.btn-secondary[href*="//"]:not([href*="api.mozillapulse.org"])```
to 
```.btn-secondary.btn-outbound-link```

And when we get to the entry page ticket we can "add" the glyph to the "Get invovled" button by assigning the right classes to it (e.g., `btn btn-secondary btn-outbound-link`)

Reply:
Okay, perfect. Thanks!